constants/const.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2022.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package constants
+
+const (
+	// ComponentName just a name to identify this package/component/application
+	ComponentName       = "kogito-builder"
+	KanikoVersion       = "1.9.0"
+	KanikoExecutorImage = "gcr.io/kaniko-project/executor:v" + KanikoVersion
```

Comment:
Are we using these constants here?

Reply:
Was used for the secret, but can be dropped for this first version